src/util.js

```diff
@@ -0,0 +1,135 @@
+const fs = require("fs");
+
+const randomColor = () => {
+  const colors = ["green", "yellow", "blue", "red", "white", "pink"];
+  const randomColorsIndex = Math.floor(Math.random() * colors.length);
+
+  return colors[randomColorsIndex];
+};
+
+const title = (answers) => {
+  return `
+  # ${answers.firstSet.title} ![${
+    answers.lastSet.license[0]
+  }](https://img.shields.io/static/v1?label=${
+    answers.lastSet.license[0]
+  }&message=License&color=${randomColor()})`;
+};
+
+const tableOfContents = (answers) => {
+  return `
+## Table of Contents 
+
+- [Description](#description)
+${
+  answers.secondSet.installationProcess ? "- [Installation](#installation)" : ""
+}
+${answers.secondSet.usageProcess ? "- [Usage](#usage)" : ""}
+${answers.secondSet.testsProcess ? "- [Tests](#tests)" : ""}
+- [Contribution](#contribution)
+- [License](#license)
+- [Contact](#contact)`;
+};
+
+const description = (answers) => {
+  return `
+## Description
+
+${answers.firstSet.description}`;
+};
+
+const installation = (answers) => {
+  return ` 
+## Installation
+  
+Run the following script to install the packages required for the application:
+
+\`\`\`
+${answers.secondSet.installationProcess}
+\`\`\``;
+};
+
+const usage = (answers) => {
+  return `
+## Usage
+
+To use the application run the following script:
+  
+\`\`\`
+${answers.secondSet.usageProcess}
+\`\`\``;
+};
+
+const tests = (answers) => {
+  return `
+## Tests
+  
+To use the application run the following script:
+  
+\`\`\`
+${answers.secondSet.testsProcess}
+\`\`\`
+  `;
+};
+
+const contributors = (answers) => {
+  return `
+## Contribution
+    
+${answers.lastSet.contribution}`;
+};
+
+const license = (answers) => {
+  return `
+## License
+
+${answers.lastSet.license[1]}
+  `;
+};
+
+const contact = (answers) => {
+  return `
+## Contact 
+
+If you have any Questions contact me using the following:
+
+- GITHUB: https://github.com/${answers.lastSet.gitHubUsername}
+
+- EMAIL: ${answers.lastSet.email}
+  `;
+};
+
+const generateReadmeFileCode = (answers) => {
```

Comment:
Destructure your answers object here so that we can send just the required data to each of the subsequent functions:

```
const {firstSet, secondSet, lastSet } = answers;

const { title, description, //other keys here } = firstSet;
const { //keys needed from secondSet } = secondSet;
const { //keys needed from lastSet } = lastSet;
```

Then just passed the required keys to the functions:

`${title(title, licence)}`

Obviously then some formatting of the functions will be required, but it will avoid all the repeating of `answers.firstSet.title` etc you have above.